### ISSUE TITLE
Turn off update nag for Firefox Nightly?

### ISSUE THREAD

**Issue by Pneumaticat** [mozilla/nixpkgs-mozilla#118]
The built-in firefox updater gives an error when trying to auto-update, which makes sense as Firefox Nightly in this repo is packaged by Nix and read-only.

Is there a way to disable it?

**Comment by zimbatm** [comment]
See https://www.technipages.com/enable-disable-automatic-updates-in-firefox

It would be cool to ship firefox with a custom prefs.js that turns this off automatically.

**Comment by garbas** [comment]
this should be done in nixpkgs, since we use `pkgs.firefox-bin-unwrapped.override` to package firefox nightly

**Comment by garbas** [comment]
i actually think i did this at one point.

**Comment by zimbatm** [comment]
Fix in https://github.com/NixOS/nixpkgs/pull/47330

**Comment by zimbatm** [comment]
This behaviour will be fixed in 18.09 and unstable as soon as the channels update. http://howoldis.herokuapp.com/

**Comment by arximboldi** [comment]
I am on 18.09 with latest nixpkgs-mozilla and I am having this problem. Did I do something wrong?
I installed nightly by adding something like this to the `configuration.nix`:
```nix
let
  mozilla = import (fetchFromGitHub {
    owner  = "mozilla";
    repo   = "nixpkgs-mozilla";
    rev    = "0d64cf67dfac2ec74b2951a4ba0141bc3e5513e8";
    sha256 = "0ngj2rk898rq73rq2rkwjax9p34mjlh3arj8w9npwwd6ljncarmh";
  });
in
  environment.systemPackages = with pkgs; [
    latest.firefox-nightly-bin
  ]
```
